example/ios/Podfile.lock

```diff
@@ -698,4 +698,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 0797f914e7e74ff6b39321a6487e474265d77004
 
-COCOAPODS: 1.11.2
+COCOAPODS: 1.15.2
```

Comment:
I don't know if it should be updated or not

Reply:
@michal-prawda @jakubsta I think it's OK